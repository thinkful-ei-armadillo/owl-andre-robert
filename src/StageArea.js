import React, { Component } from 'react';
//import './ParticipantList.css';
//import Participant from './Participant';

class StageArea extends Component {
  render() {
    return (
      <section className="stage-area">
        {/* {this.props.participantsOnStage.map((p, index) => <Participant key={index} />)} */}
      </section>
    );
  }
}

export default StageArea;
