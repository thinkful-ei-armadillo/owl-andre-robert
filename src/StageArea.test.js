import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import StageArea from './StageArea';
import {STORE} from './store';

describe('Testing stage area', () => {
  // smoke test
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<StageArea></StageArea>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  //snapshot test
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<StageArea></StageArea>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});