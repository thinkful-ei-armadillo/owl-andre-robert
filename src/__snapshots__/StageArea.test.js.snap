// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing stage area renders the UI as expected 1`] = `
<section
  className="stage-area"
>
  <div
    className="stage-participant"
  >
    <h3>
      Koren Templeton
    </h3>
    <img
      alt="avatar of user"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
  </div>
  <div
    className="stage-participant"
  >
    <h3>
      Caty Flucker
    </h3>
    <img
      alt="avatar of user"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
  </div>
  <div
    className="stage-participant"
  >
    <h3>
      Axe Kubicka
    </h3>
    <img
      alt="avatar of user"
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
  </div>
  <div
    className="stage-participant"
  >
    <h3>
      Frank Runciman
    </h3>
    <img
      alt="avatar of user"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
  </div>
  <div
    className="stage-participant"
  >
    <h3>
      Ashla Attwool
    </h3>
    <img
      alt="avatar of user"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
  </div>
</section>
`;
